### PR TITLE
Added --insecure into Export Utils

### PR DESCRIPTION
**Pull request by SafarMirek**
This PR adds --insecure parameter into Confluent Export Util and also into Apicurio Registry V1 Export Util to allow insecure HTTPS connections.

`--insecure` This option makes the client to trust all SSL certificates. 

This allows users to use self-signed certs.

### CODE REVIEW

**Comment by EricWittmann** [comment]
LGTM @carlesarnal can you also sign off?

File: utils/exportConfluent/src/main/java/io/apicurio/registry/utils/export/Export.java [hunk 1]
```diff
@@ -193,11 +194,25 @@ public int run(String... args) throws Exception {
 
             zip.flush();
             zip.close();
+        } catch (Exception ex) {
+            System.out.println("Export was not successful.");
+            ex.printStackTrace();
```
Comment (carlesarnal) [inline utils/exportConfluent/src/main/java/io/apicurio/registry/utils/export/Export.java:199]: Don't print stacktraces. Create a logger and log the exception properly.

File: utils/exportConfluent/src/main/java/io/apicurio/registry/utils/export/Export.java [hunk 2]
```diff
@@ -193,11 +194,25 @@ public int run(String... args) throws Exception {
...
         return 0;
     }
 
+    public SSLSocketFactory getInsecureSSLSocketFactory() {
+        try {
+            SSLContext sslContext = SSLContext.getInstance("SSL");
+            sslContext.init(null, new TrustManager[]{new FakeTrustManager()}, new SecureRandom());
+            return sslContext.getSocketFactory();
+        } catch (Exception ex) {
+            ex.printStackTrace();
```
Comment (carlesarnal) [inline utils/exportConfluent/src/main/java/io/apicurio/registry/utils/export/Export.java:213]: Same here

**Review by carlesarnal — CHANGES_REQUESTED** [review]
Overall lgtm, just some minor comments. Since this is a Quarkus app, you should not use System.print, you should create a proper logger and log the exceptions as required.

**Review by EricWittmann — APPROVED** [review]

**Pull request merged**